### ISSUE TITLE
time parsed error when make rows.Scan after stdlib SQL query with Go SDK

### ISSUE THREAD

**Issue by winterpi** [codenotary/immudb#1238]
**What happened**
Time value parsed incorrect:  "54379-11-09 02:20:17.42 +0000 UTC"

**What you expected to happen**
Should return readable time value, such as "2022-05-30 xx:xx:xx"

**How to reproduce it (as minimally and precisely as possible)**
1. db := stdlib.OpenDB(opts)
2. res, err := db.ExecContext(ctx, "CREATE TABLE IF NOT EXISTS table(id INTEGER AUTO_INCREMENT, time TIMESTAMP, PRIMARY KEY id)")
3. rows, err := db.QueryContext(ctx, "select * from table")
4. 	var id uint64
	var time time.Time
	rows.Next()
	err = rows.Scan(&id, &time)
       if err != nil {
		panic(err)
	}
	fmt.Println("id: ", id, ", time: ", time)

5. the result is : "id:  1 , time:  54379-11-09 02:20:17.42 +0000 UTC"

**Additional info (any other context about the problem)**
with immudb v1.3.0

**Comment by winterpi** [comment]
I found that it's not OK for TIMESTAMP when using stdlib or generic SQL.

I debugged and found a strange problem as follow:
(1). When a TIMESTAMP was inserted, it will call TimeToInt64() function in 
 embedded/sql/timestamp.go to convert timestamp to int64, e.g. time.Now() --> 1654504712965537;
(2) However, when it been saved into db, I got the value of  1654504712965537000 with sqlQuery.  So it rendered error when make schema.RenderValue(1654504712965537000).
